css/DayPickerNavigation.scss

```diff
@@ -0,0 +1,96 @@
+@import 'variables';
+
+.DayPicker__nav--prev,
+.DayPicker__nav--next {
+  cursor: pointer;
+  line-height: 0.78;
+  user-select: none;
+  -webkit-user-select: none; /* Chrome/Safari */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* IE10+ */
+}
+
+.DayPicker__nav-prev-default,
+.DayPicker__nav-next-default {
+  border: 1px solid $react-dates-color-border-light;
+  background-color: $react-dates-color-white;
+  color: $react-dates-color-placeholder-text;
+
+  &:focus,
+  &:hover {
+    border: 1px solid $react-dates-color-border-medium;
+  }
+
+  &:active {
+    background: darken($react-dates-color-white, 5%);
+  }
+}
+
+.DayPicker--horizontal {
+  .DayPicker__nav {
+    position: relative;
+  }
+
+  .DayPicker__nav--prev,
+  .DayPicker__nav--next {
+    border-radius: 3px;
+    padding: 6px 9px;
+    top: 18px;
+    z-index: 2;
+    position: absolute;
+  }
+
+  .DayPicker__nav--prev {
+    left: 22px;
+  }
+
+  .DayPicker__nav--next {
+    right: 22px;
+  }
+
+  .DayPicker__nav-next-default,
+  .DayPicker__nav-prev-default {
+    svg {
+      height: 19px;
+      width: 19px;
+      fill: $react-dates-color-gray-light;
+    }
+  }
+}
+
+.DayPicker--vertical {
+  .DayPicker__nav {
+    background: $react-dates-color-white;
+    box-shadow: 0 0 5px 2px rgba(0, 0, 0, 0.1);
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    height: 52px;
+    width: 100%;
+    z-index: 2;
+  }
+
+  .DayPicker__nav--prev,
+  .DayPicker__nav--next {
+    text-align: center;
+    font-size: 2.5em;
+    padding: 5px;
+    display: inline-block;
+    position: relative;
+    height: 100%;
+    width: 50%;
+  }
+
+  .DayPicker__nav--next {
+    border-left: 0;
```

Comment:
I'm not sure how this affect positioning, but I'll def see how it looks without.